docs/api/event-brokers.rst

```diff
@@ -29,35 +43,28 @@ tracker looks like this:
 The ``event`` field takes the event's ``type_name`` (for more on event
 types, check out the :ref:`events` docs).
 
-Rasa enables three possible broker types:
-
-- `Pika Event Broker`_
-- `Kafka Event Broker`_
-- `SQL Event Broker`_
 
 .. _event-brokers-pika:
 
 Pika Event Broker
 -----------------
 
-The example implementation we're going to show you here uses `Pika <https://pika.readthedocs.io>`_ ,
+The example implementation we're going to show you here uses `Pika <https://pika.readthedocs.io>`_,
 the Python client library for `RabbitMQ <https://www.rabbitmq.com>`_.
 
+.. contents::
+   :local:
+
 Adding a Pika Event Broker Using the Endpoint Configuration
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You can use an endpoint configuration file to instruct Rasa Core to stream
-all events to your event broker. To do so, add the following section to your
-endpoint configuration, e.g. ``endpoints.yml``:
+You can instruct Rasa to stream all events to your Pika event broker by adding an ``event_broker`` section to your
+``endpoints.yml``:
 
 .. literalinclude:: ../../data/test_endpoints/event_brokers/pika_endpoint.yml
 
-Then instruct Rasa Core to use the endpoint configuration and Pika producer by adding
-``--endpoints <path to your endpoint configuration`` as following example:
+Rasa will automatically start streaming events when you restart the Rasa server.
 
-.. code-block:: shell
-
-    rasa run -m models --endpoints endpoints.yml
 
 Adding a Pika Event Broker in Python
```

Comment:
These python additions look v out of date, why does this one pull from rasa platform code?